docs/api.rst

```diff
@@ -1064,6 +1064,25 @@ Recover from a stream failure by calling a function to create a new :ref:`Stream
 
 When ``s`` fails with an error, ``f`` will be called with the error. ``f`` must return a new :ref:`Stream` to replace the error.
 
+Sharing Streams
+^^^^^^^^^^^^^^^
+
+.. _multicast:
+
+multicast
+`````````
+
+.. code-block:: haskell
+
+  multicast :: Stream a -> Stream a
+
+Returns a :ref:`Stream` equivalent to the original, but which can be shared more efficiently among multiple consumers. ::
```

Comment:
Corrrect:
> Returns a :ref:`Stream` equivalent to the original but which can be shared more efficiently among multiple consumers.

We don’t have two independent clauses, so no comma before _but_. Even without _but_, we have an adjective clause that modifies _Stream_; _which_ is a relative pronoun referring to the antecedent _Stream_.